code/drasil-code/lib/Language/Drasil/Code/Imperative/Modules.hs

```diff
@@ -422,9 +423,9 @@ genInputFormat s = do
         outs <- getInputFormatOuts
         bod <- readData dd
         desc <- inFmtFuncDesc
-        mthd <- getFunc s "get_input" desc ins outs bod
+        mthd <- getFunc s (showChsStr GetInput) desc ins outs bod
```

Comment:
same as earlier: shouldn't `getFunc` be upgraded to have its 2nd argument be renderable?  Note that you can make String renderable with an identity function.

Reply:
Noted that this is similar to your feedback on the proposed changes to FunctionCalls.hs. I will leave my related question for that feedback as I would appreciate some clarification on what is meant by renderable in these cases. Could you please briefly note or link to some information or code that helps explain 'identity functions'? A search of the repo for those keywords did not immediately provide any leads.

Reply:
I mean
```haskell
instance RenderChoicesStr String where
  showChsStr = id
```